tests/full_cases/log-0.3.4-0.3.8.linux

```diff
@@ -71,18 +71,6 @@ warning: technically breaking changes in `<new::LogMetadata<'a> as std::fmt::Deb
     |
     = note: trait impl generalized or newly added (technically breaking)
 
-warning: technically breaking changes in `<new::NopLogger as new::Log>`
```

Comment:
This seems to have been a false positive.

Reply:
Indeed, that path looks somewhat odd.